css/properties/inline-size.json

```diff
@@ -90,11 +64,11 @@
               },
               "firefox": {
                 "prefix": "-moz-",
-                "version_added": "38"
+                "version_added": "41"
```

Comment:
This change is necessary to pass the lint, and was verified by testing https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9139 in Firefox 40 and 41.